website/server/controllers/api-v3/challenges.js

```
@@ -410,13 +541,17 @@ api.exportChallengeCsv = {
 };
 
 /**
- * @api {put} /api/v3/challenges/:challengeId Update a challenge
+ * @api {put} /api/v3/challenges/:challengeId Update a challenge. Only reflects
```

I don't think this is true. According to the group model, you should be able to update `name`, `description`, `tasksOrder`, `leader`. Is that not the case?

---

@crookedneighbor Good catch. I've added `description` and `leader`, but I'm having trouble with `tasksOrder`. I'm not sure if I should be sending a task id or an object that contains the attributes to create a new task. I've tried both with no success. I'm sure it's probably the format of the request, but since it fails silently (updating only the valid fields), I can't figure out what I'm missing. Would appreciate any help you can provide.

---

I might be wrong about task order. I'll have to look into it more.